tests/build/jvm-build.go

```diff
@@ -35,6 +35,10 @@ var (
 )
 
 var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", Label("jvm-build", "HACBS"), Pending, func() {
+	// TODO investigate and fix
```

Comment:
jvm-build-service tests are disabled anyway (using the `Pending` label)

should we enable and fix them as a part of this PR or rather do it [separately](https://issues.redhat.com/browse/PLNSRVCE-836)